modules/phase_field/src/kernels/GrainRigidBodyMotionBase.C

```diff
@@ -12,7 +12,7 @@ InputParameters validParams<GrainRigidBodyMotionBase>()
   InputParameters params = validParams<Kernel>();
   params.addClassDescription("Base class for adding rigid mody motion to grains");
   params.addRequiredCoupledVar("c", "Concentration");
-  params.addRequiredCoupledVar("v", "Array of coupled variable names");
+  params.addRequiredCoupledVarWithAutoBuild("v", "var_name_base", "op_num", "Array of coupled variable names");
```

Comment:
Does this work with `var_name_base` being a vector of strings?! I'd think it only takes the first supplied string. I'll check.